### PR TITLE
Add Superposition Testnet

### PR DESCRIPTION
**Pull request by af-afk**
### Description

Include Superposition Testnet network.

#### Other notes

Does not include any deployments.

### Testing

No

### CODE REVIEW

File: chains/superpositiontestnet/icon.svg
Comment (jmrossy) [inline chains/superpositiontestnet/icon.svg:1]: Would you mind minifying the svg image: https://jakearchibald.github.io/svgomg/

File: chains/superpositiontestnet/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
```
Comment (jmrossy) [inline chains/superpositiontestnet/metadata.yml:2]: the isTestnet field should be set to true

**Comment by af-afk** [comment]
@jmrossy Thanks for your feedback. Let me know if these changes are in line with what you were looking for.

File: chains/superpositiontestnet/metadata.yml
Comment (jmrossy) [inline chains/superpositiontestnet/metadata.yml:1]: This needs to be metadata.yaml for the tests to pass :)

File: chains/superpositiontestnet/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - apiUrl: https://testnet-explorer.superposition.so/api
+    family: blockscout
+    name: CatScan
+    url: https://testnet-explorer.superposition.so/
```
Comment (jmrossy) [inline chains/superpositiontestnet/metadata.yml:6]: Definitely not a big deal but if you don't mind, please remove the trailing slash `/`

**Comment by jmrossy** [comment]
@af-afk Thanks for the fixes! Sorry but a few more tweaks are needed

**Comment by af-afk** [comment]
> @af-afk Thanks for the fixes! Sorry but a few more tweaks are needed

No problem at all. Let me know if you need anything else now.

File: chains/superpositiontestnet/icon.svg
Comment (paulbalaji) [inline chains/superpositiontestnet/icon.svg:1]: rename to logo.svg

**Comment by af-afk** [comment]
Hi @paulbalaji, I saw you included the code. Let me know if you need me to reopen and make changes in line with your request.

**Comment by paulbalaji** [comment]
please make the suggested fixes and we can merge this, I will pull in your changes to my branch after

**Comment by paulbalaji** [comment]
please pull the latest changes from main in order to merge

**Pull request merged**